app/models/job.rb

```diff
@@ -1,6 +1,15 @@
 class Job < ActiveRecord::Base
   extend Enumerize
-  STATUSES = [:draft, :published, :archived]
-  enumerize :state, in: STATUSES
+  STATES = %w(draft published archived)
+  enumerize :state, in: STATES
   belongs_to :organization
+  validates :title,
+            presence: true,
+            length: { maximum: MAX_STRING_COLUMN_LENGTH }
```

Comment:
What are peoples thoughts on having lengths? I don't normally have them, PG and all..